src/Compatibility/Microsoft.DotNet.PackageValidation/ApiCompatRunnerExtensions.cs

```diff
@@ -2,20 +2,54 @@
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
 using Microsoft.DotNet.ApiCompatibility.Abstractions;
+using Microsoft.DotNet.ApiCompatibility.Logging;
+using Microsoft.DotNet.ApiCompatibility.Runner;
 using NuGet.ContentModel;
 using NuGet.Frameworks;
 
 namespace Microsoft.DotNet.PackageValidation
 {
     internal static class ApiCompatRunnerExtensions
     {
-        public static void QueueApiCompatFromContentItem(this ApiCompatRunner apiCompatRunner, string packageId, ContentItem leftItem, ContentItem rightItem, string header, bool isBaseline = false)
+        public static void QueueApiCompatFromContentItem(this IApiCompatRunner apiCompatRunner,
```

Comment:
Can we add tripple slash docs for this type? For example, it would be good to mention that the reason why these methods are defined here as opposed to on ApiCompatRunner directly is that this includes package-specific logic that we don't want to push to the commonly used `Microsoft.DotNet.ApiCompatibility`

Reply:
This is an internal only type which we don't triple slash document as they don't appear in the compiler produced xml file. I think it makes sense to add a comment indicating why this type exists but I wouldn't want to comment every possible method parameter. I will document the type itself.

Reply:
That's fine by me. I usually go with triple slash comments as that also helps with intellisense for the callers (even though they are internal) all relevant information is right there for them. But the primarily ask is for our code to be documented in a way that it facilitates reading the code from an external's perspective, and for that purpose, I think adding any sort of documentation to most of the new code is always valuable.

Reply:
Absolutely agree on that. It wasn't obvious to me when I first looked into this codebase why this extension class existed. A code comment would have helped. In general there were many places that would have benefited from more inline comments.